saleor/graphql/shop/mutations.py

```diff
@@ -299,6 +303,44 @@ def update_category_settings(site_settings: "SiteSettings", cleaned_input: dict)
             )
 
 
+class CategorySettingsReorderAttributes(BaseReorderAttributesMutation):
+    category_settings = graphene.Field(
+        CategorySettings, description="Reordered category settings."
+    )
+
+    class Meta:
+        description = "Reorder the category settings attributes."
+        permissions = (SitePermissions.MANAGE_SETTINGS,)
```

Comment:
Hm, I think for category attributes management we need to use different permission. I think the best one here is `MANAGE_PAGE_TYPES_AND_ATTRIBUTES`, since the attributes that we assign here are the same we assign to page types, so I assume it can by handled by the same person.

Reply:
I changed it.